crates/transaction-pool/src/pool/pending.rs

```diff
@@ -135,14 +137,16 @@ impl<T: TransactionOrdering> PendingPool<T> {
     pub(crate) fn best_with_unlocked(
         &self,
         unlocked: Vec<Arc<ValidPoolTransaction<T::Transaction>>>,
-        base_fee: u64,
-    ) -> BestTransactions<T> {
+        origin_base_fee: u64,
+        attributes: Option<BestTransactionsAttributes>,
```

Comment:
I find the optional arg a bit weird, can we make this a dedicated function instead